src/index.html

```diff
@@ -591,6 +591,66 @@ <h4>React Remixed</h4>
                 </div>
               </article>
             </li>
+            <li>
+              <article class="talks-box">
+                <img
+                  class="talks-box__image"
+                  src="images/speakers/2022/speaker-przemyslaw-beigert.png"
+                  alt="Photo of speaker: Przemysław Beigert"
+                />
+                <div class="talks-box__info">
+                  <div class="talks-box__header">
+                    <div>
+                      <h3>Przemysław Beigert</h3>
+                      <a
+                        href="https://twitter.com/przemyslawjanp"
+                        target="_blank"
+                        title="Twitter profile - Przemysław Beigert"
+                        >#przemyslawjanp</a
+                      >
+                    </div>
+                  </div>
+
+                  <h4>Advanced TypeScript</h4>
+                  <p class="talks-box__desc">
+                    TypeScript become extremely popular tool but how well you know him? Probably you
+                    can write and read code in it, but many features seems to be useless, right? In
+                    my talk I'll present less known functions of this language and explain how you
+                    can use them in your project to increase type safeness.
+                  </p>
+                </div>
+              </article>
+            </li>
+            <li>
+              <article class="talks-box">
+                <img
+                  class="talks-box__image"
+                  src="images/speakers/2022/speaker-konrad-hanus.png"
+                  alt="Photo of speaker: Konrad Hanus"
+                />
+                <div class="talks-box__info">
+                  <div class="talks-box__header">
+                    <div>
+                      <h3>Konrad Hanus</h3>
+                      <a
+                        href="https://twitter.com/KonradHanus"
+                        target="_blank"
+                        title="Twitter profile - Konrad Hanus"
+                        >#KonradHanus</a
+                      >
+                    </div>
+                  </div>
+
+                  <h4>Our approach to unifying company products with shareable library</h4>
```

Comment:
How to make an engine for a side scroller video game using HTML5 Cavnas and React